include/utility/dbus_utility.hpp

```diff
@@ -231,5 +231,31 @@ inline bool isServiceRunning(const std::string& i_serviceName)
 
     return l_retVal;
 }
+
+inline types::BiosAttributeCurrentValue
+    biosGetAttributeMethodCall(const std::string& i_attributeName)
+{
+    auto l_bus = sdbusplus::bus::new_default();
+    auto l_method = l_bus.new_method_call(
+        constants::biosConfigMgrService, constants::biosConfigMgrObjPath,
+        constants::biosConfigMgrInterface, "GetAttribute");
+    l_method.append(i_attributeName);
+
+    types::BiosGetAttrRetType l_attributeVal;
+    try
+    {
+        auto l_result = l_bus.call(l_method);
+        l_result.read(std::get<0>(l_attributeVal), std::get<1>(l_attributeVal),
+                      std::get<2>(l_attributeVal));
+    }
+    catch (const sdbusplus::exception::SdBusError& l_ex)
+    {
+        logging::logMessage(
+            "Failed to read BIOS Attribute: " + i_attributeName +
+            " due to error " + std::string(l_ex.what()));
+    }
+
+    return std::get<1>(l_attributeVal);
```

Comment:
does this return empty tuple if exception caught ?
did not get the info from oxygen so asking

Reply:
Do we need an informational log since we have not backed up the data ?

Reply:
> does this return empty tuple if exception caught

Ooops, missed doxygen here.

> Do we need an informational log since we have not backed up the data

Currently we don't, should we? SSR would be interested in this info or journal should be ok?